examples/angular/src/app/app.component.ts

```diff
@@ -59,7 +63,7 @@ export class AppComponent implements OnInit {
           },
         }),
         setupNightlyConnect({
-          url: "wss://ncproxy.nightly.app/app",
```

Comment:
Why have you changed this?

Reply:
Sorry my mistake not synching to latest dev. Just fixed it